lib/graphql_rails/model/configurable.rb

```diff
@@ -5,6 +5,14 @@ module Model
     # contains methods which are shared between various configurations
     # expects `default_name` to be defined
     module Configurable
+      def initialize_copy(other)
+        super
+        @name = nil
+        @type_name = nil
+        @description = nil
+        @attributes = other.instance_variable_get(:@attributes)&.transform_values(&:dup)
```

Comment:
why do you need to access it through an instance variable? `@attributes` has a public reader